CHANGELOG.rst

```diff
@@ -43,6 +43,9 @@ Changelog
   on LibreSSL.
 * Added support for RSA PSS signatures in PKCS7 with
   :meth:`~cryptography.hazmat.primitives.serialization.pkcs7.PKCS7SignatureBuilder.add_signer`.
+* In the next release (43.0.0) of cryptography, loading an X.509 certificate
+  with a negative serial number will raise an exception. This has been
```

Comment:
43.0.0 was released, but no mention of this. See https://github.com/pyca/cryptography/issues/10247#issuecomment-2242549924